include/coreir/ir/value.h

```diff
@@ -206,9 +194,6 @@ class TemplatedConst : public Const {
     bool operator<(const Value& r) const override;
 
     static bool classof(const Value* v) {return v->getKind()==Underlying2Kind<T>::kind;}
-    //static std::shared_ptr<TemplatedConst<T>> make(ValueType* type, T value) {
-    //  return std::make_shared<TemplatedConst<T>>(type,value);
-    //}
```

Comment:
Not sure why the two above methods are removed.

Reply:
This was never in the code base. It was always commented. Probably should have removed it in a different PR